src/common/events_v2/native_event.hpp

```diff
@@ -244,4 +245,78 @@ namespace events_comm
 
     friend class TrEventDetailStorage;
   };
+
+  /**
+   * Input event types for keyboard, mouse, and other input events
+   */
+  enum class TrInputEventType
+  {
+    Unknown = 0,
+    KeyboardDown,
+    KeyboardUp,
+    KeyboardPress,
+    MouseDown,
+    MouseUp,
+    MouseMove,
```

Comment:
MouseMove can't be exist at this project, because we don't allow content receive the native window's mouse move event, as for MouseDown and MouseUp, it has been handled by WebXR input source such as primary actions, so you need to remove them here, too.